osmosis-apidb/src/main/java/org/openstreetmap/osmosis/apidb/v0_6/ApidbWriter.java

```diff
@@ -394,24 +394,22 @@ private void initialize() {
      * @param node The node containing the data to be inserted.
      */
     private void populateNodeParameters(PreparedStatement statement, int initialIndex, Node node) {
-        int prmIndex;
```

Comment:
This is ignoring the initialIndex parameter which will cause prmIndex to be initialised incorrectly.  The initialIndex parameter can't be ignored.